CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Mattermost Desktop Application Changelog
 
+## Release v3.4.1
+
+Release date: TBD
+
+### Bug Fixes
+
+#### macOS
```

Comment:
I think we don't have to do that because the name was "OS X" when releasing past versions.
In this part, I had thought OS name like [Electron's changelog](https://github.com/electron/electron/releases/tag/v1.4.2).